### ISSUE TITLE
Winetricks returned status 67

### ISSUE THREAD

**Issue by yifongau** [rockerbacon/lutris-skyrimse-installers#114]
Hi, I have an issue where I cannot finish the installation. I get the message that it is unable to install winetricks, but I already have winetricks installed. The terminal logs the following at the end of the process:

```
------------------------------------------------------
Note: command wine dotNetFx40_Full_x86_x64.exe /q /c:install.exe /q returned status 67. Aborting.
------------------------------------------------------
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 256
2020-07-15 16:07:04,805: I can't move /home/yifongau/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
2020-07-15 16:07:04,806: Error while completing task <bound method CommandsMixin.move of <lutris.installer.interpreter.ScriptInterpreter object at 0x7fab8e086d68>>: I can't move /home/yifongau/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
<class 'lutris.installer.errors.ScriptingError'> I can't move /home/yifongau/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
  File "/usr/lib/python3/dist-packages/lutris/util/jobs.py", line 34, in target
    result = self.function(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/lutris/installer/commands.py", line 277, in move
    raise ScriptingError("I can't move %s, it does not exist" % src)
2020-07-15 16:07:08,619: Cancelling installation of Mod Organizer 2
```
I have also added the full terminal logs in pastebin: https://pastebin.com/G0JdwvEY

Not sure how I should resolve this issue. Anyone have any ideas?

**Comment by rockerbacon** [comment]
Are you using Arch or Manjaro?

**Comment by rockerbacon** [comment]
This should have been fixed in #116

**Comment by yifongau** [comment]
> Are you using Arch or Manjaro?

Thanks for responding so quickly. Regarding your question: Neither. I am using debian 10.
I wasn’t sure what you meant by dotnet48, but after some googling I used the command `winetricks dotnet48`, and that seemed to install dotnet48. However, after I tried the installer again it aborted with the same ‘status 67’ message. Am I missing something?
